cmd/alias-list.go

```diff
@@ -156,7 +156,7 @@ func listAliases(alias string, deprecated bool) (aliases []aliasMessage) {
 		if len(pair) != 2 {
 			continue
 		}
-		if len(strings.TrimPrefix(pair[0], mcEnvHostPrefix)) != 0 {
+		if strings.HasPrefix(pair[0], mcEnvHostPrefix) {
```

Comment:
This function is a frivolous is a better PR https://github.com/minio/mc/pull/5048